Automattic/wp-calypso
# Help Center: Change header icon style

## Proposed Changes

* Changes help center's the header Icon according to this [thread](https://href.li/?https://wp.me/pbAok1-2D2) 


New Icon 
<img width="278" alt="image" src="https://user-images.githubusercontent.com/2653810/201927361-cf294c4b-8f3e-45bf-ade0-d6f6e8e80a87.png">


Old Icon 
<img width="163" alt="image" src="https://user-images.githubusercontent.com/2653810/201927413-09d2ef82-dc45-4158-93bb-4602e63722ce.png">


## Testing Instructions
- Pull the branch
- Open [localhost](http://calypso.localhost:3000/) ( and where else the help center icon available )
- Check the header if the icon matches the design

Related to #68015
Fixes #68015
